packages/fastify-renderer/src/node/renderers/react/ReactRenderer.tsx

```diff
@@ -69,14 +58,111 @@ export class ReactRenderer implements Renderer {
       for (const renderable of renderables) {
         await this.loadModule(this.entrypointRequirePathForServer(renderable))
       }
+
+      const mode = this.options.mode
+      this.transformHooks = this.plugin.hooks
```

Comment:
Need to get paths to each transform hook module in order for the worker to load it. They are pre-loaded along with all the other bundles.